lib/internal/Magento/Framework/Filesystem/Driver/File.php

```diff
@@ -589,16 +589,24 @@ public function fileOpen($path, $mode)
      * @param int $length
      * @param string $ending [optional]
      * @return string
-     * @throws FileSystemException
+     * @throws LocalizedException
      */
+
     public function fileReadLine($resource, $length, $ending = null)
     {
-        $result = @stream_get_line($resource, $length, $ending);
-        if (false === $result) {
-            throw new FileSystemException(
-                new \Magento\Framework\Phrase('File cannot be read %1', [$this->getWarningMessage()])
+        try {
+            $result = @stream_get_line($resource, $length, $ending);
+            if (false === $result) {
+                throw new FileSystemException(
+                    new \Magento\Framework\Phrase('File cannot be read %1', [$this->getWarningMessage()])
+                );
+            }
+        } catch (\Exception $e) {
+            throw new LocalizedException(
```

Comment:
Looks like case where this exception is thrown isn’t covered , could you cover it?